grpc/server.go

```diff
@@ -0,0 +1,84 @@
+package sentrygrpc
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// UnaryServerInterceptor is a grpc interceptor that reports errors and panics
+// to sentry. It also sets *sentry.Hub to context.
+func UnaryServerInterceptor(opts UnaryServerInterceptorOptions) grpc.UnaryServerInterceptor {
```

Comment:
@shouichi do you envision there being some non-unary server interceptor later? If not, we could perhaps shorten the name to `ServerInterceptor`, so that call sites become `sentrygrpc.ServerInterceptor`.

Asking because I see in OpenTracing they name a similar interceptor [`OpenTracingServerInterceptor`](https://github.com/opentracing-contrib/go-grpc/blob/4a6760c71486b5e9aa6faf22b11d579e13a119d9/server.go#L27) (used as `otgrpc.OpenTracingServerInterceptor`, the "OpenTracing" prefix being a bit redundant).

Reply:
Honestly not in the near future (because we don't use grpc streams). I'm happy to contribute tho ;)

OpenTelemetry (which I believe a successor of OpenTracing) uses `UnaryServerInterceptor`. I personally prefer to prefix "Unary" because it is explicit and will be consistent after adding a stream interceptor.

https://github.com/open-telemetry/opentelemetry-go-contrib/blob/65367f2b373efda45942c5c37b4332a31b226b98/instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go#L304

Reply:
👍 

Indeed it shall be `UnaryServerInterceptor`, so later we can also have `StreamServerInterceptor`.